src/commcare_cloud/fab/deploy_diff.py

```diff
@@ -102,15 +102,29 @@ def render_diff(self, template_name):
             **self.get_diff_context()
         )
 
-    def _get_pr_numbers(self):
+    def _get_commits_on_master_since_last_deploy(self):
+        last_deployed_commit = self.repo.get_commit(self.current_commit)
+        return self.repo.get_commits(sha='master', since=last_deployed_commit.commit.author.date)
+
+    def _get_merge_commits_since_last_deploy(self):
         comparison = self.repo.compare(self.current_commit, self.deploy_commit)
```

Comment:
if `current_commit` is the latest commit on `master` and `deploy_commit` is the last deployed commit (do I have that right?) then I don't think I get how this could include things that aren't in the `master` branch's history. Can you explain this to me a bit more?

Reply:
The only explanation I have is the Comparison object picks up all commits in between the two specified commits by date, regardless of branch. As far as I could tell from [the docs](https://pygithub.readthedocs.io/en/latest/github_objects/Repository.html#github.Repository.Repository.compare), there isn't a way to specify a branch when retrieving a comparison between two commits.